src/linux/zygote.rs

```diff
@@ -306,8 +306,8 @@ fn timed_wait(pid: Pid, timeout: Option<time::Duration>) -> Result<Option<ExitCo
     end_w = 0;
     setup_pipe(&mut end_r, &mut end_w)?;
     let waiter_pid;
+    let mut arg = WaiterArg { res_fd: end_w, pid };
```

Comment:
```suggestion
    let mut waiter_arg = WaiterArg { res_fd: end_w, pid };
```
larger scope -> longer name :)